server/sonar-server/src/main/java/org/sonar/server/issue/IssueService.java

```diff
@@ -210,12 +137,16 @@ public Issue getByKey(String key) {
     return issueIndex.getByKey(key);
   }
 
+  // TODO Either use IssueFinder or remove it
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1135)